game/src/contrib/configuration/KeyboardConfig.java

```diff
@@ -17,6 +17,7 @@ public class KeyboardConfig {
             new ConfigKey<>(new String[] {"movement", "left"}, new ConfigIntValue(Input.Keys.A));
     public static final ConfigKey<Integer> MOVEMENT_RIGHT =
             new ConfigKey<>(new String[] {"movement", "right"}, new ConfigIntValue(Input.Keys.D));
+
```

Comment:
?

Reply:
was hast du hier genau anzumerken?

Reply:
Ich denke, es geht darum, dass in der Datei nichts verändert, aber eine Leerzeile eingefügt wurde.